inst/Upgrades.qml

```diff
@@ -0,0 +1,171 @@
+import QtQuick		2.12
+import JASP.Module	1.0
+
+Upgrades
+{
+	Upgrade
+	{
+		functionName:		"NetworkAnalysis"
+		fromVersion:		"0.15"
+		toVersion:			"0.16.4"
+
+		ChangeJS
+		{
+			name: "estimator"
+			jsFunction: function(options)
+			{
+				switch (options["estimator"])
+				{
+					case "EBICglasso":			return "ebicGlasso";
+					case "IsingFit":			return "isingFit";
+					case "IsingSampler":		return "isingSampler";
+					// the other values did not change
+					default:					return options["estimator"];
+
+				}
+			}
+		}
+
+		ChangeRename	{	from:	"plotNetwork";						to:		"networkPlot"									}
+		ChangeRename	{	from:	"plotCentrality";					to:		"centralityPlot"								}
+		ChangeRename	{	from:	"plotClustering";					to:		"networkclusteringPlotPlot"						}
+
+		ChangeRename	{	from: 	"tableCentrality";					to:		"centralityTable"								}
+		ChangeRename	{	from: 	"tableClustering";					to:		"clusteringTable"								}
+		ChangeRename	{	from: 	"tableWeightsMatrix";				to:		"weightsMatrixTable"							}
+
+		ChangeRename	{	from:	"normalizeCentrality";				to:		"centralityNormalization"						}
+
+		ChangeRename	{	from: 	"mgmVariableTypeContinuous";		to:		"mgmContinuousVariables"						}
+		ChangeRename	{	from: 	"mgmVariableTypeCategorical";		to:		"mgmCategoricalVariables"						}
+		ChangeRename	{	from: 	"mgmVariableTypeCount";				to:		"mgmCountVariables"								}
+
+		ChangeRename	{	from:	"BootstrapType";					to:		"bootstrapType"									}
+		ChangeRename	{	from:	"bootstrapOnOff";					to:		"bootstrap"										}
+		ChangeRename	{	from:	"numberOfBootstraps";				to:		"bootstrapSamples"								}
+		ChangeRename	{	from:	"parallelBootstrap";				to:		"bootstrapParallel"								}
+		
+
+		ChangeRename	{	from: 	"StatisticsEdges";					to:		"statisticsEdges"								}
+		ChangeRename	{	from: 	"StatisticsCentrality";				to:		"statisticsCentrality"							}
+
+		ChangeRename	{	from: 	"groupNames";						to:		"manualColorGroups"								}
+
+		ChangeJS
+		{
+			name: "manualColorGroups"
+			jsFunction: function(options)
+			{
+
+				options["manualColorGroups"] = options["manualColorGroups"].forEach(item=>{
+					item["name"] = item["group"];
+					item["color"] = item["groupColor"];
+					return item;
+				})
+			}
+		}
+		
+		ChangeRename	{	from:	"variablesForColor";				to:		"colorGroupVariables"							}
+		ChangeJS
+		{
+			name: "colorGroupVariables"
+			jsFunction: function(options)
+			{
+
+				options["colorGroupVariables"] = options["colorGroupVariables"].forEach(item=>{
+					item["group"] = item["groupAssigned"];
+					return item;
+				})
```

Comment:
There is something wrong in the upgrade of `variablesForColor` -> `colorGroupVariables` (and its suboptions) as the information seems to get lost:

in 0.16.3
![Screenshot 2022-08-05 at 10 35 56](https://user-images.githubusercontent.com/7093546/183037831-1a50844a-582e-484a-a57b-19bbae16f163.png)

opened with this version
![Screenshot 2022-08-05 at 10 36 06](https://user-images.githubusercontent.com/7093546/183037848-e4e8ed5a-8b80-49cc-b848-c8fbb19ad4ad.png)

Interestingly, `manualColorGroups` (formerly `groupNames`) upgrades work with the same syntax:

in 0.16.3
![Screenshot 2022-08-05 at 10 38 57](https://user-images.githubusercontent.com/7093546/183038382-ca4b8c76-f333-4e11-b0ad-3c478f482754.png)

opened with this version
![Screenshot 2022-08-05 at 10 39 21](https://user-images.githubusercontent.com/7093546/183038500-46f7e44d-7fb5-4314-9474-4f1fe664df24.png)

Here is the jasp file made in 0.16.3 that I used for testing
[network_0.16.3.jasp.zip](https://github.com/jasp-stats/jaspNetwork/files/9266993/network_0.16.3.jasp.zip)

Reply:
Except for this, it seems everything else works

Reply:
Actually, I think they were both broken because I saw null in the json for both fields. The screenshot shows the default value. It should be working now though.